Makefile

```
@@ -16,10 +16,13 @@ validate-schemas:
 	./scripts/validate_schemas.sh
 
 translate-schemas:
+	pipenv run python -m scripts.eq_translations_check
 	pipenv run python -m scripts.translate_schemas
 
 translation-templates:
+	pipenv run python -m scripts.eq_translations_check
 	pipenv run python -m scripts.extract_translation_templates
 
 test-translation-templates:
+	pipenv run python -m scripts.eq_translations_check
 	pipenv run python -m scripts.extract_translation_templates --test
```

Not a blocker but reduces reference that may change.
```suggestion
eq_translations_check:
	pipenv run python -m scripts.eq_translations_check

translate-schemas: eq_translations_check
	pipenv run python -m scripts.translate_schemas

translation-templates: eq_translations_check
	pipenv run python -m scripts.extract_translation_templates

test-translation-templates: eq_translations_check
	pipenv run python -m scripts.extract_translation_templates --test
```

---

Added, thanks!👍